visidata/_types.py

```diff
@@ -53,7 +54,7 @@ def getType(typetype):
 vdtype(anytype, '', formatter=lambda _,v: str(v))
 vdtype(str, '~', formatter=lambda _,v: v)
 vdtype(int, '#', '%.0f')
-vdtype(float, '%', '%.02f')
+vdtype(float, '%', formatter=lambda _,v: options.disp_float_fmt % v)
```

Comment:
almost there...the formatter should be like the disp_date_fmt usage:

`vdtype(date, '@', '', formatter=lambda fmtstr,val: val.strftime(fmtstr or options.disp_date_fmt))`

The first parameter is the given fmtstr, as set by the user on the Column sheet.  So we have to take that into account similarly with disp_float_fmt or the user won't be able to override fmtstr on float columns.